packaging/package.csproj

```diff
@@ -0,0 +1,64 @@
+<Project Sdk="Microsoft.Build.NoTargets/3.5.6">
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <IsPackable>true</IsPackable>
+        <NoWarn>NU5100;NU5128</NoWarn>
+        <VSCompatTargetFramework>net472</VSCompatTargetFramework>
+
+        <!-- MSBuild tasks shouldn't be referenced. This is by design. -->
+        <NoWarn>NU5100;NU5128</NoWarn>
+        <Authors>Microsoft</Authors>
+        <Company>Microsoft</Company>
+        <Packageid>Microsoft.NET.Build.Containers</Packageid>
```

Comment:
Is this case-sensitive? might need to be `PackageId`.

Reply:
It isn't, but it should follow whatever the convention is. Updated